src/socket.c

```diff
@@ -2655,18 +2637,15 @@ int handle_fake_recv_function(conString *string, const char *world,
     }
     if (raw)
     {
-	handle_socket_input(string->data, string->len);
+	handle_socket_input(string->data, string->len, NULL);
     }
     else
-    {
-        queue_socket_line(sock, string, 0, 0);
-        flushxsock();
-    }
+	queue_socket_line(sock, string, string->len, 0);
     return 1;
```

Comment:
Should be fine to add this back in as well

Reply:
If its needed can you restore it?